src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -152,6 +159,19 @@ public synchronized void deleteCard(Card card) throws CardNotFoundException {
         indicateAddressBookChanged();
     }
 
+    @Override
+    public void answerSelectedCard(int confidenceLevel) throws NoCardSelectedException {
+        if (selectedCard == null) {
+            throw new NoCardSelectedException();
+        } else if (confidenceLevel == 0) {
+            // repeat in ui
```

Comment:
This can be done by shifting the position of the card to the back of the list and then calling showDueCards

Reply:
I was thinking that the UI side would have something to handle viewing of cards.

But sure this works too thanks!